### PR TITLE
Feature lockers

### PR DESCRIPTION
**Pull request by ImplicitDeny**
Basically rework locker functions and move them to GSRI Gear mod. Once merged, we will be able to [remove them from GSRI Opex](https://github.com/team-gsri/GSRI-Opex/issues/74).

### CODE REVIEW

**Comment by ImplicitDeny** [comment]
Once merged, will close #21

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -83,6 +83,14 @@ jobs:
         includes: '*.paa;*.xml'
         flags: '"-clear","-prefix=fr\gsri\vests"'
 
+    - name: Build GSRI Lockers
+      uses: team-gsri/actions-addon-builder@master
+      with:
+        source: 'addons/gsri_lockers'
+        target: 'addons'
+        includes: '*.paa;*.xml'
```
Comment (ArwynFr) [inline .github/workflows/release.yml:91]: sqf ?

Comment (ImplicitDeny) [reply .github/workflows/release.yml:91]: Shouldn't we standardize this kind of thing once and for all, with a proper inclusion list ? I'd rather have an optional exclusion list than a mandatory inclusion list to modify on each and every mod. I don't see the added value here ?

Comment (ArwynFr) [reply .github/workflows/release.yml:91]: sorry, this is how addon builder works :(
you can use `*.*` but you will have both config.cpp and config.bin in the pbo.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -83,6 +83,14 @@ jobs:
         includes: '*.paa;*.xml'
         flags: '"-clear","-prefix=fr\gsri\vests"'
 
+    - name: Build GSRI Lockers
+      uses: team-gsri/actions-addon-builder@master
+      with:
+        source: 'addons/gsri_lockers'
+        target: 'addons'
+        includes: '*.paa;*.xml'
+        flags: '"-clear","-prefix=fr\gsri\lockers"'
```
Comment (ArwynFr) [inline .github/workflows/release.yml:92]:
```suggestion
    - name: Build GSRI Lockers
      uses: team-gsri/actions-build-addon@0.1.1
      with:
        source: 'addons/gsri_lockers'
        target: 'addons'
        prefix: 'fr\gsri\lockers'

```

**Review by ArwynFr — DISMISSED** [review]
LGTM

**Review by ArwynFr — APPROVED** [review]

**Pull request merged**